src/System.Private.CoreLib/src/System/Globalization/FormatProvider.FormatAndParse.Unix.cs

```diff
@@ -0,0 +1,164 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime;
+using System.Globalization;
+using System.Diagnostics;
+
+namespace System.Globalization
+{
+    internal partial class FormatProvider
+    {
+        private partial class Number
+        {
+            // buffer size to hold digits (40), decimal point, number sign, exponent, exponent symbol 'e', exponent sign and null.
+            private const int  ECVT_MAX_BUFFER_SIZE = 50;  
```

Comment:
We are not calling ecvt method anymore here - we should pick a different name for it.